include/dnnl.hpp

```diff
@@ -10555,6 +10555,10 @@ enum class cpu_isa {
     avx512_core_bf16 = dnnl_cpu_isa_avx512_core_bf16,
     /// @copydoc dnnl_cpu_isa_avx512_core_amx
     avx512_core_amx = dnnl_cpu_isa_avx512_core_amx,
+    /// @copydoc dnnl_cpu_isa_simdfp
+    simdfp = dnnl_cpu_isa_simdfp,
```

Comment:
I'm not sure simdfp is the best name to use - see later comments.

Reply:
I changed `simdfp` -> `asimd`.
https://github.com/oneapi-src/oneDNN/commit/26a9f0a1b5d524f4abc1c5a748011bcf9d8f2038

Reply:
ack.